docs/spec/spec.md

```diff
@@ -58,33 +70,49 @@ broker._
 
 ## kind: Broker
 
-### group: eventing.knative.dev/v1beta1
+### group: eventing.knative.dev/v1
 
-_A Broker represents an event mesh. It logically receives events on its input
-domain and forwards them to subscribers defined by one or more matching
-Trigger._
+_A `Broker` collects a pool of events that are consumable using triggers.
+Brokers provide a well-known endpoint for event delivery that senders can use
+with minimal knowledge of the event routing strategy. Subscribers use triggers
+to request delivery of events from a broker's pool to a specific URL or
+`Addressable` endpoint._
 
 ### Object Schema
 
+#### Metadata
+
+Standard Kubernetes
+[meta.v1/ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta)
+resource.
+
 #### Spec
 
-| Field    | Type                             | Description                                                                                                                                                  | Constraints |
-| -------- | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
-| config   | ObjectReference                  | Config is an ObjectReference to the configuration that specifies configuration options for this Broker. For example, this could be a pointer to a ConfigMap. |             |
-| delivery | eventingduckv1beta1.DeliverySpec | Delivery is the delivery specification for Events within the Broker mesh. This includes things like retries, DLQ, etc.                                       |             |
+The `BrokerSpec` defines the desired state for the `Broker`.
+
+| Field Name | Field Type                                                              | Requirement | Description                                                                                                  | Constraints |
+| ---------- | ----------------------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------ | ----------- |
+| `config`   | [`duckv1.KReference`](#duckv1.kreference)                               | Optional    | Reference to the configuration options for this broker. For example, this could be a pointer to a ConfigMap. |             |
+| `delivery` | [`eventingduckv1beta1.DeliverySpec`](#eventingduckv1beta1.deliveryspec) | Optional    | The delivery specification for events within the broker mesh. This includes things like retries, DLQ, etc.   |             |
```

Comment:
What's "broker mesh"?

Reply:
The current descriptions are based on the existing spec and the apis with some occasional wordsmithing. The broker was frequently described as an ["event mesh"](https://knative.dev/v0.16-docs/eventing/broker/#broker). I am not a fan of the use of the word "mesh" here either, so I'm happy to cut it out.

Reply:
I would vote for that. Broker mesh sounds like a mesh of brokers, it seems that this can be "a broker" or "brokers".